demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -261,3 +265,44 @@ def test_generate_commands_with_permissions_section():
         '##### Human Readable Output', '', '']
 
     assert '\n'.join(section) == '\n'.join(expected_section)
+
+
+class TestAppendOrReplaceCommandInDocs:
+    positive_test_data_file = os.path.join(FILES_PATH, 'docs_test', 'positive_docs_section_end_with_eof.md')
+    command = 'dxl-send-event'
+    old_doc = open(positive_test_data_file).read()
+    new_docs = "\n<NEW DOCS>\n"
```

Comment:
not sure i understand this test, why do we need that new docs?? why not just test the actual functionality with readme files being edited with either append or replace?

Reply:
to see if the append worked and save properly in the generate_integration_docs command.
and we need to raise the coverage.

Reply:
the new docs is a doc section in the new test.

Reply:
i understand what it is, what i dont understand is why you chose to test it like that..
lets test with sections we have in our docs

Reply:
because it's much easier to maintain.
It'll replace an actual section with a string that is not important at all.

Reply:
maintain? whats there to maintain? its just a simple test..
i want it to test real input..

Reply:
it'll not change the behavior of the test in any means.